Frameworks/Plugins/ZappCrashlogsMsAppCenter/Manifest/ios.json

```diff
@@ -4,16 +4,19 @@
     "class_name": "ZappCrashlogsMsAppCenter.ZappCrashlogsMsAppCenter",
     "modules": []
   },
-  "dependency_repository_url": [],
+  "dependency_name": "@applicaster/crash-logs-ms-app-center",
+  "npm_dependencies": [
+    "@applicaster/crash-logs-ms-app-center@<%= version_id %>"
+  ],
   "extra_dependencies": [
     {
-      "ZappCrashlogsMsAppCenter": ":git => 'https://github.com/applicaster/AppleApplicasterFrameworks.git', :tag => 2020.56.5.15-1-5"
+      "ZappCrashlogsMsAppCenter": ":path => './node_modules/crash-logs-ms-app-center/ZappCrashlogsMsAppCenter.podspec'"
     }
   ],
   "platform": "ios",
   "author_name": "Anton Kononenko",
   "author_email": "a.kononenko@applicaster.com",
-  "manifest_version": "4.2.0",
+  "manifest_version": "4.2.1",
```

Comment:
is this kept in sync with the npm version of the package ? it's not mandatory but I think it's easier to reason with. Since the manifest is updated by a script, it should be possible to have it with a variable like `<%= version_id %>` above